ui/de-lib/src/main/java/org/iplantc/de/apps/widgets/client/view/editors/arguments/converters/ArgumentEditorConverter.java

```diff
@@ -142,14 +142,11 @@ public void applyValidators(List<ArgumentValidator> validators, boolean isValida
 
                 }
 
-                // If validation is not disabled, Re-apply the actual validator
-                if (!isValidationDisabled) {
-                    Object validator = autoBean.getTag(ArgumentValidator.VALIDATOR);
-                    if ((field instanceof Field<?>) && (validator != null)) {
-                        @SuppressWarnings("unchecked")
-                        Validator<T> val = (Validator<T>)validator;
-                        ((Field<T>)field).addValidator(val);
-                    }
+                // Re-apply the actual validator
+                Object validator = autoBean.getTag(ArgumentValidator.VALIDATOR);
+                if ((field instanceof Field<?>) && (validator != null)) {
+                    @SuppressWarnings("unchecked") Validator<T> val = (Validator<T>)validator;
+                    ((Field<T>)field).addValidator(val);
```

Comment:
Yes, just getting rid of the check for `isValidationDisabled` because it's always set to disabled prior to this method getting called and then re-enabled afterwards.  From what I could tell, we disabled validations prior to adding any validators otherwise as soon as the app was launched, every single field with validators would immediately have the red box around it instead of validating after input was entered.